src/components/organisms/forms/JourneyForm/JourneyForm.component.tsx

```diff
@@ -0,0 +1,74 @@
+'use client';
+import IconCard, { IconCardProps } from '@/components/molecules/cards/IconCard';
+import FormFooter from '@/components/organisms/FormFooter/FormFooter.component';
+import { FormActionConfig } from '@/components/organisms/forms/FormAction';
+import { Container } from '@mui/material';
+import { useRouter } from 'next/navigation';
+import { useEffect, useMemo, useState } from 'react';
+
+export interface JourneyFormProps {
+  choices: Record<string, IconCardProps & { route: string }>;
+}
+
+const JourneyForm = ({ choices }: JourneyFormProps) => {
+  const router = useRouter();
+  const [choice, setChoice] = useState<string | null>(null);
+
+  // PREFETCH LINKED PAGES
+  useEffect(() => {
+    Object.keys(choices).forEach((c) => {
+      router.prefetch(choices[c].route);
+    });
+  }, [router, choices]);
+
+  const nextButtonConfig: FormActionConfig = useMemo(
+    () => ({
+      hidden: false,
+      isLoading: false,
+      disabled: choice === null,
+      text: 'Next',
+      onClick: () => {
+        if (choice) router.push(choices[choice].route);
+      },
```

Comment:
Nit: I wonder if we can add an `href` prop to the `FormActionConfig` that can be used if it's just a classic route change. That would keep things more accessible

Reply:
and then this would just be
```suggestion
      href: choices[choice]?.route,
```